react-app/src/components/ServerFormModal/index.js

```diff
@@ -86,29 +90,29 @@ export default function ServerFormModal({ }) {
 
                     <label for='server-form-server-image'>Enter a Server Image Url</label>
                     <input
-                        type='text'
+                        type='file'
                         id='server-form-server-image'
                         name='server-form-server-image'
                         required
-                        value={serverImage}
-                        onChange={(e) => setServerImage(e.target.value)}
+                        onChange={(e) => setServerImage(e.target.files[0])}
+                        accept='.jpg, .jpeg, .png'
```

Comment:
Is there a reason that this "accept" attribute is being limited to `'.jpg, .jpeg, .png'` instead of `'image/*'` like the others?

Reply:
This section has been updated to match the others.